src/os/ui/slick/slickgrid.js

```diff
@@ -1259,6 +1269,47 @@ os.ui.slick.SlickGridCtrl.prototype.apply = function() {
 };
 
 
+/**
+ * A given cell's editor has been created
+ *
+ * @protected
+ */
+os.ui.slick.SlickGridCtrl.prototype.beforeCellEdited = function() {
+  this.scope.$emit(os.ui.slick.SlickGridEvent.BEFORE_CELL_EDITED, arguments);
+};
+
+
+/**
+ * A given cell's value has been changed
+ *
+ * @protected
+ */
+os.ui.slick.SlickGridCtrl.prototype.cellChanged = function() {
+  this.scope.$emit(os.ui.slick.SlickGridEvent.CELL_CHANGED, arguments);
+};
+
+
+/**
+ * The cell is no longer being edited.
+ * This is triggered when an edit has been made or discarded.
+ *
+ * @protected
+ */
+os.ui.slick.SlickGridCtrl.prototype.cellEditorDestroyed = function() {
+  this.scope.$emit(os.ui.slick.SlickGridEvent.CELL_EDITOR_DESTROYED, arguments);
```

Comment:
Not sure how the compiler missed it, but `arguments` isn't defined

Reply:
`arguments` is available on any function that is not an arrow function. See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments) for details.

It does, however, skip typing the function parameters in the compiler, which is not so great. This is completely valid JS, but the typing for the closure compiler could be better.

Reply:
The other Slickgrid listeners we define in here all just have a vaguely defined `args` object that gets passed to them from the Slickgrid notify calls. It's only marginally better, but these should follow that pattern if possible.

Reply:
I pushed a few changes to improve how the listener functions are defined.